Test failure in CI build 1216

The following test appears to have failed:

[#1216](https://circleci.com/gh/cockroachdb/cockroach/1216):

```
I0408 15:41:10.875698     271 multiraft.go:650] Outgoing Message[0]: 514->257 MsgAppResp Term:6 Log:0/20 Rejected
I0408 15:41:10.876167     271 multiraft.go:633] node 257: group 1 raft ready
I0408 15:41:10.876339     271 multiraft.go:650] Outgoing Message[0]: 257->514 MsgHeartbeat Term:6 Log:0/0
I0408 15:41:10.877085     271 multiraft.go:820] node 514: connecting to new node 257
W0408 15:41:10.877210     271 multiraft.go:828] node 514 failed to send message to 257
--- FAIL: TestFailedReplicaChange (0.16s)
	<autogenerated>:31: Test appears to have leaked an rpc client:
		net.(*pollDesc).Wait(0xc208677800, 0x72, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:84 +0x63
		net.(*pollDesc).WaitRead(0xc208677800, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:89 +0x51
		net.(*netFD).Read(0xc2086777a0, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x7f9dba0e1e20, 0xc208681018)
			/usr/src/go/src/net/fd_unix.go:242 +0x4b3
		net.(*conn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0xc208607640, 0x0, 0x0)
			/usr/src/go/src/net/net.go:121 +0x125
		net.(*TCPConn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
--
I0408 15:41:11.144248     271 multiraft.go:448] node 257: group 1 got message 514->257 MsgAppResp Term:6 Log:0/18
I0408 15:41:11.144312     271 multiraft.go:633] node 514: group 1 raft ready
I0408 15:41:11.144390     271 multiraft.go:650] Outgoing Message[0]: 514->257 MsgHeartbeatResp Term:6 Log:0/0
W0408 15:41:11.144805     271 multiraft.go:828] node 514 failed to send message to 257
E0408 15:41:11.145367     271 multiraft.go:530] node 514: error sending coalesced heartbeat to 257: transport.go:160: unknown peer 257
--- FAIL: TestReplicateAfterTruncation (0.27s)
	<autogenerated>:31: Test appears to have leaked an rpc client:
		net.(*pollDesc).Wait(0xc208677800, 0x72, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:84 +0x63
		net.(*pollDesc).WaitRead(0xc208677800, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:89 +0x51
		net.(*netFD).Read(0xc2086777a0, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x7f9dba0e1e20, 0xc208681018)
			/usr/src/go/src/net/fd_unix.go:242 +0x4b3
		net.(*conn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0xc208607640, 0x0, 0x0)
			/usr/src/go/src/net/net.go:121 +0x125
		net.(*TCPConn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
--
I0408 15:41:11.901558     271 multiraft.go:633] node 257: group 1 raft ready
I0408 15:41:11.901691     271 multiraft.go:650] Outgoing Message[0]: 257->771 MsgHeartbeat Term:14 Log:0/0 Commit:19
I0408 15:41:11.901792     271 multiraft.go:650] Outgoing Message[1]: 257->514 MsgHeartbeat Term:14 Log:0/0 Commit:19
E0408 15:41:11.902943     271 multiraft.go:530] node 771: error sending coalesced heartbeat to 257: transport.go:160: unknown peer 257
E0408 15:41:11.903097     271 multiraft.go:530] node 771: error sending coalesced heartbeat to 514: transport.go:160: unknown peer 514
--- FAIL: TestStoreRangeReplicate (0.75s)
	<autogenerated>:31: Test appears to have leaked an rpc client:
		net.(*pollDesc).Wait(0xc208677800, 0x72, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:84 +0x63
		net.(*pollDesc).WaitRead(0xc208677800, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:89 +0x51
		net.(*netFD).Read(0xc2086777a0, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x7f9dba0e1e20, 0xc208681018)
			/usr/src/go/src/net/fd_unix.go:242 +0x4b3
		net.(*conn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0xc208607640, 0x0, 0x0)
			/usr/src/go/src/net/net.go:121 +0x125
		net.(*TCPConn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
--
			/go/src/github.com/cockroachdb/cockroach/storage/main_test.go:29 +0x36
		main.main()
			github.com/cockroachdb/cockroach/storage/_test/_testmain.go:332 +0x28d
=== RUN TestSetupRangeTree
I0408 15:41:11.985804     271 multiraft.go:407] node 257 starting
--- FAIL: TestSetupRangeTree (0.09s)
	<autogenerated>:31: Test appears to have leaked an rpc client:
		net.(*pollDesc).Wait(0xc208677800, 0x72, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:84 +0x63
		net.(*pollDesc).WaitRead(0xc208677800, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:89 +0x51
		net.(*netFD).Read(0xc2086777a0, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x7f9dba0e1e20, 0xc208681018)
			/usr/src/go/src/net/fd_unix.go:242 +0x4b3
		net.(*conn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0xc208607640, 0x0, 0x0)
			/usr/src/go/src/net/net.go:121 +0x125
		net.(*TCPConn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
--
I0408 15:41:12.524544     271 retry.go:93] Get failed; retrying immediately
I0408 15:41:12.524931     271 multiraft.go:633] node 257: group 4 raft ready
I0408 15:41:12.525052     271 multiraft.go:638] HardState updated: {Term:6 Vote:257 Commit:24 XXX_unrecognized:[]}
I0408 15:41:12.525898     271 multiraft.go:641] New Entry[0]: 6/24 EntryNormal 0000000000000000678877f0fa6cb0f2: raft_id:4 cmd:<internal_resolve_intent:<header:<timestamp:<wall_time:0 logical:157 > cmd_id:<wall_time:0 random:0 > key:"\000\000\000kc\000\001rtn-" 
I0408 15:41:12.526670     271 multiraft.go:644] Committed Entry[0]: 6/24 EntryNormal 0000000000000000678877f0fa6cb0f2: raft_id:4 cmd:<internal_resolve_intent:<header:<timestamp:<wall_time:0 logical:157 > cmd_id:<wall_time:0 random:0 > key:"\000\000\000kc\000\001rtn-" 
--- FAIL: TestInsertRight (0.54s)
	<autogenerated>:31: Test appears to have leaked an rpc client:
		net.(*pollDesc).Wait(0xc208677800, 0x72, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:84 +0x63
		net.(*pollDesc).WaitRead(0xc208677800, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:89 +0x51
		net.(*netFD).Read(0xc2086777a0, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x7f9dba0e1e20, 0xc208681018)
			/usr/src/go/src/net/fd_unix.go:242 +0x4b3
		net.(*conn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0xc208607640, 0x0, 0x0)
			/usr/src/go/src/net/net.go:121 +0x125
		net.(*TCPConn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
--
I0408 15:41:13.121805     271 retry.go:93] Get failed; retrying immediately
I0408 15:41:13.122294     271 multiraft.go:633] node 257: group 5 raft ready
I0408 15:41:13.122397     271 multiraft.go:638] HardState updated: {Term:6 Vote:257 Commit:17 XXX_unrecognized:[]}
I0408 15:41:13.123321     271 multiraft.go:641] New Entry[0]: 6/17 EntryNormal 00000000000000003c00c9f3537cf162: raft_id:5 cmd:<internal_resolve_intent:<header:<timestamp:<wall_time:0 logical:166 > cmd_id:<wall_time:0 random:0 > key:"\000\000\000kb\000\001rtn-" 
I0408 15:41:13.124119     271 multiraft.go:644] Committed Entry[0]: 6/17 EntryNormal 00000000000000003c00c9f3537cf162: raft_id:5 cmd:<internal_resolve_intent:<header:<timestamp:<wall_time:0 logical:166 > cmd_id:<wall_time:0 random:0 > key:"\000\000\000kb\000\001rtn-" 
--- FAIL: TestInsertLeft (0.60s)
	<autogenerated>:31: Test appears to have leaked an rpc client:
		net.(*pollDesc).Wait(0xc208677800, 0x72, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:84 +0x63
		net.(*pollDesc).WaitRead(0xc208677800, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:89 +0x51
		net.(*netFD).Read(0xc2086777a0, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x7f9dba0e1e20, 0xc208681018)
			/usr/src/go/src/net/fd_unix.go:242 +0x4b3
		net.(*conn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0xc208607640, 0x0, 0x0)
			/usr/src/go/src/net/net.go:121 +0x125
		net.(*TCPConn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
--
			/go/src/github.com/cockroachdb/cockroach/storage/main_test.go:29 +0x36
		main.main()
			github.com/cockroachdb/cockroach/storage/_test/_testmain.go:332 +0x28d
=== RUN TestStoreRangeSplitAtIllegalKeys
I0408 15:41:13.206884     271 multiraft.go:407] node 257 starting
--- FAIL: TestStoreRangeSplitAtIllegalKeys (0.08s)
	<autogenerated>:31: Test appears to have leaked an rpc client:
		net.(*pollDesc).Wait(0xc208677800, 0x72, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:84 +0x63
		net.(*pollDesc).WaitRead(0xc208677800, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:89 +0x51
		net.(*netFD).Read(0xc2086777a0, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x7f9dba0e1e20, 0xc208681018)
			/usr/src/go/src/net/fd_unix.go:242 +0x4b3
		net.(*conn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0xc208607640, 0x0, 0x0)
			/usr/src/go/src/net/net.go:121 +0x125
		net.(*TCPConn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
--
I0408 15:41:13.340130     271 multiraft.go:638] HardState updated: {Term:6 Vote:257 Commit:21 XXX_unrecognized:[]}
I0408 15:41:13.341222     271 multiraft.go:641] New Entry[0]: 6/21 EntryNormal 13d314290b7ba0955acaa5c0c26b4e61: raft_id:1 cmd:<end_transaction:<header:<timestamp:<wall_time:0 logical:8 > cmd_id:<wall_time:1428507673325445269 random:6542223656023182945 > key:"a"
I0408 15:41:13.342231     271 multiraft.go:644] Committed Entry[0]: 6/21 EntryNormal 13d314290b7ba0955acaa5c0c26b4e61: raft_id:1 cmd:<end_transaction:<header:<timestamp:<wall_time:0 logical:8 > cmd_id:<wall_time:1428507673325445269 random:6542223656023182945 > key:"a"
I0408 15:41:13.346643     271 raft.go:390] raft: 101 became follower at term 5
I0408 15:41:13.346908     271 raft.go:207] raft: newRaft 101 [peers: [101], term: 5, commit: 10, applied: 10, lastindex: 10, lastterm: 5]
--- FAIL: TestStoreRangeSplitAtRangeBounds (0.15s)
	<autogenerated>:31: Test appears to have leaked an rpc client:
		net.(*pollDesc).Wait(0xc208677800, 0x72, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:84 +0x63
		net.(*pollDesc).WaitRead(0xc208677800, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:89 +0x51
		net.(*netFD).Read(0xc2086777a0, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x7f9dba0e1e20, 0xc208681018)
			/usr/src/go/src/net/fd_unix.go:242 +0x4b3
		net.(*conn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0xc208607640, 0x0, 0x0)
			/usr/src/go/src/net/net.go:121 +0x125
		net.(*TCPConn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
--
I0408 15:41:13.519997     271 multiraft.go:644] Committed Entry[1]: 6/23 EntryNormal 00000000000000001448c1cb2ccf3d4a: raft_id:1 cmd:<internal_resolve_intent:<header:<timestamp:<wall_time:0 logical:17 > cmd_id:<wall_time:0 random:0 > key:"\000\000\000k\000\001rtn-" us
I0408 15:41:13.520839     271 multiraft.go:644] Committed Entry[2]: 6/24 EntryNormal 00000000000000007689dc7a74866ea3: raft_id:1 cmd:<internal_resolve_intent:<header:<timestamp:<wall_time:0 logical:17 > cmd_id:<wall_time:0 random:0 > key:"\000\000meta1\377\377" user:"
I0408 15:41:13.521748     271 multiraft.go:644] Committed Entry[3]: 6/25 EntryNormal 000000000000000079180663d178adeb: raft_id:1 cmd:<internal_resolve_intent:<header:<timestamp:<wall_time:0 logical:17 > cmd_id:<wall_time:0 random:0 > key:"\000\000meta2a" user:"root" r
I0408 15:41:13.522449     271 multiraft.go:644] Committed Entry[4]: 6/26 EntryNormal 00000000000000004e25865f44fa13d1: raft_id:1 cmd:<internal_resolve_intent:<header:<timestamp:<wall_time:0 logical:17 > cmd_id:<wall_time:0 random:0 > key:"\000\000meta2\377\377" user:"
I0408 15:41:13.523301     271 multiraft.go:644] Committed Entry[5]: 6/27 EntryNormal 0000000000000000131339e37bcd83b5: raft_id:1 cmd:<internal_resolve_intent:<header:<timestamp:<wall_time:0 logical:17 > cmd_id:<wall_time:0 random:0 > key:"\000range-tree-root" user:"ro
--- FAIL: TestStoreRangeSplitConcurrent (0.17s)
	<autogenerated>:31: Test appears to have leaked an rpc client:
		net.(*pollDesc).Wait(0xc208677800, 0x72, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:84 +0x63
		net.(*pollDesc).WaitRead(0xc208677800, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:89 +0x51
		net.(*netFD).Read(0xc2086777a0, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x7f9dba0e1e20, 0xc208681018)
			/usr/src/go/src/net/fd_unix.go:242 +0x4b3
		net.(*conn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0xc208607640, 0x0, 0x0)
			/usr/src/go/src/net/net.go:121 +0x125
		net.(*TCPConn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
--
I0408 15:41:13.674280     271 multiraft.go:638] HardState updated: {Term:6 Vote:257 Commit:25 XXX_unrecognized:[]}
I0408 15:41:13.675222     271 multiraft.go:641] New Entry[0]: 6/25 EntryNormal 13d314291f9f4fff3940d667a3aa3ac2: raft_id:1 cmd:<end_transaction:<header:<timestamp:<wall_time:0 logical:12 > cmd_id:<wall_time:1428507673663328255 random:4125532999287192258 > key:"m
I0408 15:41:13.676168     271 multiraft.go:644] Committed Entry[0]: 6/25 EntryNormal 13d314291f9f4fff3940d667a3aa3ac2: raft_id:1 cmd:<end_transaction:<header:<timestamp:<wall_time:0 logical:12 > cmd_id:<wall_time:1428507673663328255 random:4125532999287192258 > key:"m
I0408 15:41:13.680524     271 raft.go:390] raft: 101 became follower at term 5
I0408 15:41:13.680748     271 raft.go:207] raft: newRaft 101 [peers: [101], term: 5, commit: 10, applied: 10, lastindex: 10, lastterm: 5]
--- FAIL: TestStoreRangeSplit (0.17s)
	<autogenerated>:31: Test appears to have leaked an rpc client:
		net.(*pollDesc).Wait(0xc208677800, 0x72, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:84 +0x63
		net.(*pollDesc).WaitRead(0xc208677800, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:89 +0x51
		net.(*netFD).Read(0xc2086777a0, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x7f9dba0e1e20, 0xc208681018)
			/usr/src/go/src/net/fd_unix.go:242 +0x4b3
		net.(*conn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0xc208607640, 0x0, 0x0)
			/usr/src/go/src/net/net.go:121 +0x125
		net.(*TCPConn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
--
I0408 15:41:14.216927     271 multiraft.go:638] HardState updated: {Term:6 Vote:257 Commit:118 XXX_unrecognized:[]}
I0408 15:41:14.217937     271 multiraft.go:641] New Entry[0]: 6/118 EntryNormal 13d314293ff7114479edf21af6548515: raft_id:2 cmd:<end_transaction:<header:<timestamp:<wall_time:0 logical:239 > cmd_id:<wall_time:1428507674205950276 random:8785944645685511445 > key:
I0408 15:41:14.218865     271 multiraft.go:644] Committed Entry[0]: 6/118 EntryNormal 13d314293ff7114479edf21af6548515: raft_id:2 cmd:<end_transaction:<header:<timestamp:<wall_time:0 logical:239 > cmd_id:<wall_time:1428507674205950276 random:8785944645685511445 > key:
I0408 15:41:14.229301     271 raft.go:390] raft: 101 became follower at term 5
I0408 15:41:14.229524     271 raft.go:207] raft: newRaft 101 [peers: [101], term: 5, commit: 10, applied: 10, lastindex: 10, lastterm: 5]
--- FAIL: TestStoreRangeSplitStats (0.56s)
	<autogenerated>:31: Test appears to have leaked an rpc client:
		net.(*pollDesc).Wait(0xc208677800, 0x72, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:84 +0x63
		net.(*pollDesc).WaitRead(0xc208677800, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:89 +0x51
		net.(*netFD).Read(0xc2086777a0, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x7f9dba0e1e20, 0xc208681018)
			/usr/src/go/src/net/fd_unix.go:242 +0x4b3
		net.(*conn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0xc208607640, 0x0, 0x0)
			/usr/src/go/src/net/net.go:121 +0x125
		net.(*TCPConn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
--
I0408 15:41:15.313958     271 multiraft.go:638] HardState updated: {Term:6 Vote:257 Commit:270 XXX_unrecognized:[]}
I0408 15:41:15.314795     271 multiraft.go:641] New Entry[0]: 6/269 EntryNormal 0000000000000000249c4c120ad92b6f: raft_id:1 cmd:<internal_resolve_intent:<header:<timestamp:<wall_time:0 logical:495 > cmd_id:<wall_time:0 random:0 > key:"\000\000meta2\377\377" user
I0408 15:41:15.315594     271 multiraft.go:641] New Entry[1]: 6/270 EntryNormal 00000000000000003682bdd11f910f6a: raft_id:1 cmd:<internal_resolve_intent:<header:<timestamp:<wall_time:0 logical:495 > cmd_id:<wall_time:0 random:0 > key:"\000range-tree-root" user:"
I0408 15:41:15.316341     271 multiraft.go:644] Committed Entry[0]: 6/269 EntryNormal 0000000000000000249c4c120ad92b6f: raft_id:1 cmd:<internal_resolve_intent:<header:<timestamp:<wall_time:0 logical:495 > cmd_id:<wall_time:0 random:0 > key:"\000\000meta2\377\377" user
I0408 15:41:15.317144     271 multiraft.go:644] Committed Entry[1]: 6/270 EntryNormal 00000000000000003682bdd11f910f6a: raft_id:1 cmd:<internal_resolve_intent:<header:<timestamp:<wall_time:0 logical:495 > cmd_id:<wall_time:0 random:0 > key:"\000range-tree-root" user:"
--- FAIL: TestStoreZoneUpdateAndRangeSplit (1.07s)
	<autogenerated>:31: Test appears to have leaked an rpc client:
		net.(*pollDesc).Wait(0xc208677800, 0x72, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:84 +0x63
		net.(*pollDesc).WaitRead(0xc208677800, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:89 +0x51
		net.(*netFD).Read(0xc2086777a0, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x7f9dba0e1e20, 0xc208681018)
			/usr/src/go/src/net/fd_unix.go:242 +0x4b3
		net.(*conn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0xc208607640, 0x0, 0x0)
			/usr/src/go/src/net/net.go:121 +0x125
		net.(*TCPConn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
--
I0408 15:41:16.055371     271 multiraft.go:641] New Entry[1]: 6/86 EntryNormal 000000000000000075f1b42a29e00edc: raft_id:1 cmd:<internal_resolve_intent:<header:<timestamp:<wall_time:0 logical:267 > cmd_id:<wall_time:0 random:0 > key:"\000\000meta2db1" user:"root
I0408 15:41:16.055956     271 multiraft.go:641] New Entry[2]: 6/87 EntryNormal 00000000000000002b976c7e501daf7d: raft_id:1 cmd:<internal_resolve_intent:<header:<timestamp:<wall_time:0 logical:267 > cmd_id:<wall_time:0 random:0 > key:"\000\000meta2db2" user:"root
I0408 15:41:16.056583     271 multiraft.go:644] Committed Entry[0]: 6/85 EntryNormal 000000000000000060e4756f5b62bb85: raft_id:1 cmd:<internal_resolve_intent:<header:<timestamp:<wall_time:0 logical:267 > cmd_id:<wall_time:0 random:0 > key:"\000\000meta1\377\377" user:
I0408 15:41:16.057223     271 multiraft.go:644] Committed Entry[1]: 6/86 EntryNormal 000000000000000075f1b42a29e00edc: raft_id:1 cmd:<internal_resolve_intent:<header:<timestamp:<wall_time:0 logical:267 > cmd_id:<wall_time:0 random:0 > key:"\000\000meta2db1" user:"root
I0408 15:41:16.057822     271 multiraft.go:644] Committed Entry[2]: 6/87 EntryNormal 00000000000000002b976c7e501daf7d: raft_id:1 cmd:<internal_resolve_intent:<header:<timestamp:<wall_time:0 logical:267 > cmd_id:<wall_time:0 random:0 > key:"\000\000meta2db2" user:"root
--- FAIL: TestStoreRangeSplitOnConfigs (0.75s)
	<autogenerated>:31: Test appears to have leaked an rpc client:
		net.(*pollDesc).Wait(0xc208677800, 0x72, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:84 +0x63
		net.(*pollDesc).WaitRead(0xc208677800, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:89 +0x51
		net.(*netFD).Read(0xc2086777a0, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x7f9dba0e1e20, 0xc208681018)
			/usr/src/go/src/net/fd_unix.go:242 +0x4b3
		net.(*conn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0xc208607640, 0x0, 0x0)
			/usr/src/go/src/net/net.go:121 +0x125
		net.(*TCPConn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
--
I0408 15:41:16.281744     271 multiraft.go:644] Committed Entry[0]: 6/45 EntryNormal 13d31429bb4285473e2b9e5941e1fad2: raft_id:1 cmd:<put:<header:<timestamp:<wall_time:0 logical:39 > cmd_id:<wall_time:1428507676274492743 random:4479848360544697042 > key:"\000\000meta2
I0408 15:41:16.283321     271 multiraft.go:633] node 257: group 1 raft ready
I0408 15:41:16.283411     271 multiraft.go:638] HardState updated: {Term:6 Vote:257 Commit:46 XXX_unrecognized:[]}
I0408 15:41:16.283934     271 multiraft.go:641] New Entry[0]: 6/46 EntryNormal 13d31429bb42ec9c0d4476bebf1ae7ab: raft_id:1 cmd:<put:<header:<timestamp:<wall_time:0 logical:40 > cmd_id:<wall_time:1428507676274519196 random:956019582531463083 > key:"\000\000meta1\
I0408 15:41:16.284493     271 multiraft.go:644] Committed Entry[0]: 6/46 EntryNormal 13d31429bb42ec9c0d4476bebf1ae7ab: raft_id:1 cmd:<put:<header:<timestamp:<wall_time:0 logical:40 > cmd_id:<wall_time:1428507676274519196 random:956019582531463083 > key:"\000\000meta1\
--- FAIL: TestUpdateRangeAddressing (0.22s)
	<autogenerated>:31: Test appears to have leaked an rpc client:
		net.(*pollDesc).Wait(0xc208677800, 0x72, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:84 +0x63
		net.(*pollDesc).WaitRead(0xc208677800, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:89 +0x51
		net.(*netFD).Read(0xc2086777a0, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x7f9dba0e1e20, 0xc208681018)
			/usr/src/go/src/net/fd_unix.go:242 +0x4b3
		net.(*conn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0xc208607640, 0x0, 0x0)
			/usr/src/go/src/net/net.go:121 +0x125
		net.(*TCPConn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
--
			/go/src/github.com/cockroachdb/cockroach/storage/main_test.go:29 +0x36
		main.main()
			github.com/cockroachdb/cockroach/storage/_test/_testmain.go:332 +0x28d
=== RUN TestUpdateRangeAddressingSplitMeta1
I0408 15:41:16.381585     271 multiraft.go:407] node 257 starting
--- FAIL: TestUpdateRangeAddressingSplitMeta1 (0.10s)
	<autogenerated>:31: Test appears to have leaked an rpc client:
		net.(*pollDesc).Wait(0xc208677800, 0x72, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:84 +0x63
		net.(*pollDesc).WaitRead(0xc208677800, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:89 +0x51
		net.(*netFD).Read(0xc2086777a0, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x7f9dba0e1e20, 0xc208681018)
			/usr/src/go/src/net/fd_unix.go:242 +0x4b3
		net.(*conn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0xc208607640, 0x0, 0x0)
			/usr/src/go/src/net/net.go:121 +0x125
		net.(*TCPConn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
--
			/go/src/github.com/cockroachdb/cockroach/util/leaktest/leaktest.go:34 +0x36
		github.com/cockroachdb/cockroach/storage.TestMain(0xc208030b90)
			/go/src/github.com/cockroachdb/cockroach/storage/main_test.go:29 +0x36
		main.main()
			github.com/cockroachdb/cockroach/storage/_test/_testmain.go:332 +0x28d
FAIL
FAIL	github.com/cockroachdb/cockroach/storage	10.023s
=== RUN TestBatchBasics
--- PASS: TestBatchBasics (0.00s)
=== RUN TestBatchGet
--- PASS: TestBatchGet (0.00s)
=== RUN TestBatchMerge
--- PASS: TestBatchMerge (0.00s)
=== RUN TestBatchProto
--- PASS: TestBatchProto (0.00s)
=== RUN TestBatchScan
--- PASS: TestBatchScan (0.00s)
I0408 15:41:10.875698     271 multiraft.go:650] Outgoing Message[0]: 514->257 MsgAppResp Term:6 Log:0/20 Rejected
I0408 15:41:10.876167     271 multiraft.go:633] node 257: group 1 raft ready
I0408 15:41:10.876339     271 multiraft.go:650] Outgoing Message[0]: 257->514 MsgHeartbeat Term:6 Log:0/0
I0408 15:41:10.877085     271 multiraft.go:820] node 514: connecting to new node 257
W0408 15:41:10.877210     271 multiraft.go:828] node 514 failed to send message to 257
--- FAIL: TestFailedReplicaChange (0.16s)
	<autogenerated>:31: Test appears to have leaked an rpc client:
		net.(*pollDesc).Wait(0xc208677800, 0x72, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:84 +0x63
		net.(*pollDesc).WaitRead(0xc208677800, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:89 +0x51
		net.(*netFD).Read(0xc2086777a0, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x7f9dba0e1e20, 0xc208681018)
			/usr/src/go/src/net/fd_unix.go:242 +0x4b3
		net.(*conn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0xc208607640, 0x0, 0x0)
			/usr/src/go/src/net/net.go:121 +0x125
		net.(*TCPConn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
--
I0408 15:41:11.144248     271 multiraft.go:448] node 257: group 1 got message 514->257 MsgAppResp Term:6 Log:0/18
I0408 15:41:11.144312     271 multiraft.go:633] node 514: group 1 raft ready
I0408 15:41:11.144390     271 multiraft.go:650] Outgoing Message[0]: 514->257 MsgHeartbeatResp Term:6 Log:0/0
W0408 15:41:11.144805     271 multiraft.go:828] node 514 failed to send message to 257
E0408 15:41:11.145367     271 multiraft.go:530] node 514: error sending coalesced heartbeat to 257: transport.go:160: unknown peer 257
--- FAIL: TestReplicateAfterTruncation (0.27s)
	<autogenerated>:31: Test appears to have leaked an rpc client:
		net.(*pollDesc).Wait(0xc208677800, 0x72, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:84 +0x63
		net.(*pollDesc).WaitRead(0xc208677800, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:89 +0x51
		net.(*netFD).Read(0xc2086777a0, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x7f9dba0e1e20, 0xc208681018)
			/usr/src/go/src/net/fd_unix.go:242 +0x4b3
		net.(*conn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0xc208607640, 0x0, 0x0)
			/usr/src/go/src/net/net.go:121 +0x125
		net.(*TCPConn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
--
I0408 15:41:11.901558     271 multiraft.go:633] node 257: group 1 raft ready
I0408 15:41:11.901691     271 multiraft.go:650] Outgoing Message[0]: 257->771 MsgHeartbeat Term:14 Log:0/0 Commit:19
I0408 15:41:11.901792     271 multiraft.go:650] Outgoing Message[1]: 257->514 MsgHeartbeat Term:14 Log:0/0 Commit:19
E0408 15:41:11.902943     271 multiraft.go:530] node 771: error sending coalesced heartbeat to 257: transport.go:160: unknown peer 257
E0408 15:41:11.903097     271 multiraft.go:530] node 771: error sending coalesced heartbeat to 514: transport.go:160: unknown peer 514
--- FAIL: TestStoreRangeReplicate (0.75s)
	<autogenerated>:31: Test appears to have leaked an rpc client:
		net.(*pollDesc).Wait(0xc208677800, 0x72, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:84 +0x63
		net.(*pollDesc).WaitRead(0xc208677800, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:89 +0x51
		net.(*netFD).Read(0xc2086777a0, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x7f9dba0e1e20, 0xc208681018)
			/usr/src/go/src/net/fd_unix.go:242 +0x4b3
		net.(*conn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0xc208607640, 0x0, 0x0)
			/usr/src/go/src/net/net.go:121 +0x125
		net.(*TCPConn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
--
			/go/src/github.com/cockroachdb/cockroach/storage/main_test.go:29 +0x36
		main.main()
			github.com/cockroachdb/cockroach/storage/_test/_testmain.go:332 +0x28d
=== RUN TestSetupRangeTree
I0408 15:41:11.985804     271 multiraft.go:407] node 257 starting
--- FAIL: TestSetupRangeTree (0.09s)
	<autogenerated>:31: Test appears to have leaked an rpc client:
		net.(*pollDesc).Wait(0xc208677800, 0x72, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:84 +0x63
		net.(*pollDesc).WaitRead(0xc208677800, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:89 +0x51
		net.(*netFD).Read(0xc2086777a0, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x7f9dba0e1e20, 0xc208681018)
			/usr/src/go/src/net/fd_unix.go:242 +0x4b3
		net.(*conn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0xc208607640, 0x0, 0x0)
			/usr/src/go/src/net/net.go:121 +0x125
		net.(*TCPConn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
--
I0408 15:41:12.524544     271 retry.go:93] Get failed; retrying immediately
I0408 15:41:12.524931     271 multiraft.go:633] node 257: group 4 raft ready
I0408 15:41:12.525052     271 multiraft.go:638] HardState updated: {Term:6 Vote:257 Commit:24 XXX_unrecognized:[]}
I0408 15:41:12.525898     271 multiraft.go:641] New Entry[0]: 6/24 EntryNormal 0000000000000000678877f0fa6cb0f2: raft_id:4 cmd:<internal_resolve_intent:<header:<timestamp:<wall_time:0 logical:157 > cmd_id:<wall_time:0 random:0 > key:"\000\000\000kc\000\001rtn-" 
I0408 15:41:12.526670     271 multiraft.go:644] Committed Entry[0]: 6/24 EntryNormal 0000000000000000678877f0fa6cb0f2: raft_id:4 cmd:<internal_resolve_intent:<header:<timestamp:<wall_time:0 logical:157 > cmd_id:<wall_time:0 random:0 > key:"\000\000\000kc\000\001rtn-" 
--- FAIL: TestInsertRight (0.54s)
	<autogenerated>:31: Test appears to have leaked an rpc client:
		net.(*pollDesc).Wait(0xc208677800, 0x72, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:84 +0x63
		net.(*pollDesc).WaitRead(0xc208677800, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:89 +0x51
		net.(*netFD).Read(0xc2086777a0, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x7f9dba0e1e20, 0xc208681018)
			/usr/src/go/src/net/fd_unix.go:242 +0x4b3
		net.(*conn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0xc208607640, 0x0, 0x0)
			/usr/src/go/src/net/net.go:121 +0x125
		net.(*TCPConn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
--
I0408 15:41:13.121805     271 retry.go:93] Get failed; retrying immediately
I0408 15:41:13.122294     271 multiraft.go:633] node 257: group 5 raft ready
I0408 15:41:13.122397     271 multiraft.go:638] HardState updated: {Term:6 Vote:257 Commit:17 XXX_unrecognized:[]}
I0408 15:41:13.123321     271 multiraft.go:641] New Entry[0]: 6/17 EntryNormal 00000000000000003c00c9f3537cf162: raft_id:5 cmd:<internal_resolve_intent:<header:<timestamp:<wall_time:0 logical:166 > cmd_id:<wall_time:0 random:0 > key:"\000\000\000kb\000\001rtn-" 
I0408 15:41:13.124119     271 multiraft.go:644] Committed Entry[0]: 6/17 EntryNormal 00000000000000003c00c9f3537cf162: raft_id:5 cmd:<internal_resolve_intent:<header:<timestamp:<wall_time:0 logical:166 > cmd_id:<wall_time:0 random:0 > key:"\000\000\000kb\000\001rtn-" 
--- FAIL: TestInsertLeft (0.60s)
	<autogenerated>:31: Test appears to have leaked an rpc client:
		net.(*pollDesc).Wait(0xc208677800, 0x72, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:84 +0x63
		net.(*pollDesc).WaitRead(0xc208677800, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:89 +0x51
		net.(*netFD).Read(0xc2086777a0, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x7f9dba0e1e20, 0xc208681018)
			/usr/src/go/src/net/fd_unix.go:242 +0x4b3
		net.(*conn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0xc208607640, 0x0, 0x0)
			/usr/src/go/src/net/net.go:121 +0x125
		net.(*TCPConn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
--
			/go/src/github.com/cockroachdb/cockroach/storage/main_test.go:29 +0x36
		main.main()
			github.com/cockroachdb/cockroach/storage/_test/_testmain.go:332 +0x28d
=== RUN TestStoreRangeSplitAtIllegalKeys
I0408 15:41:13.206884     271 multiraft.go:407] node 257 starting
--- FAIL: TestStoreRangeSplitAtIllegalKeys (0.08s)
	<autogenerated>:31: Test appears to have leaked an rpc client:
		net.(*pollDesc).Wait(0xc208677800, 0x72, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:84 +0x63
		net.(*pollDesc).WaitRead(0xc208677800, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:89 +0x51
		net.(*netFD).Read(0xc2086777a0, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x7f9dba0e1e20, 0xc208681018)
			/usr/src/go/src/net/fd_unix.go:242 +0x4b3
		net.(*conn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0xc208607640, 0x0, 0x0)
			/usr/src/go/src/net/net.go:121 +0x125
		net.(*TCPConn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
--
I0408 15:41:13.340130     271 multiraft.go:638] HardState updated: {Term:6 Vote:257 Commit:21 XXX_unrecognized:[]}
I0408 15:41:13.341222     271 multiraft.go:641] New Entry[0]: 6/21 EntryNormal 13d314290b7ba0955acaa5c0c26b4e61: raft_id:1 cmd:<end_transaction:<header:<timestamp:<wall_time:0 logical:8 > cmd_id:<wall_time:1428507673325445269 random:6542223656023182945 > key:"a"
I0408 15:41:13.342231     271 multiraft.go:644] Committed Entry[0]: 6/21 EntryNormal 13d314290b7ba0955acaa5c0c26b4e61: raft_id:1 cmd:<end_transaction:<header:<timestamp:<wall_time:0 logical:8 > cmd_id:<wall_time:1428507673325445269 random:6542223656023182945 > key:"a"
I0408 15:41:13.346643     271 raft.go:390] raft: 101 became follower at term 5
I0408 15:41:13.346908     271 raft.go:207] raft: newRaft 101 [peers: [101], term: 5, commit: 10, applied: 10, lastindex: 10, lastterm: 5]
--- FAIL: TestStoreRangeSplitAtRangeBounds (0.15s)
	<autogenerated>:31: Test appears to have leaked an rpc client:
		net.(*pollDesc).Wait(0xc208677800, 0x72, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:84 +0x63
		net.(*pollDesc).WaitRead(0xc208677800, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:89 +0x51
		net.(*netFD).Read(0xc2086777a0, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x7f9dba0e1e20, 0xc208681018)
			/usr/src/go/src/net/fd_unix.go:242 +0x4b3
		net.(*conn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0xc208607640, 0x0, 0x0)
			/usr/src/go/src/net/net.go:121 +0x125
		net.(*TCPConn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
--
I0408 15:41:13.519997     271 multiraft.go:644] Committed Entry[1]: 6/23 EntryNormal 00000000000000001448c1cb2ccf3d4a: raft_id:1 cmd:<internal_resolve_intent:<header:<timestamp:<wall_time:0 logical:17 > cmd_id:<wall_time:0 random:0 > key:"\000\000\000k\000\001rtn-" us
I0408 15:41:13.520839     271 multiraft.go:644] Committed Entry[2]: 6/24 EntryNormal 00000000000000007689dc7a74866ea3: raft_id:1 cmd:<internal_resolve_intent:<header:<timestamp:<wall_time:0 logical:17 > cmd_id:<wall_time:0 random:0 > key:"\000\000meta1\377\377" user:"
I0408 15:41:13.521748     271 multiraft.go:644] Committed Entry[3]: 6/25 EntryNormal 000000000000000079180663d178adeb: raft_id:1 cmd:<internal_resolve_intent:<header:<timestamp:<wall_time:0 logical:17 > cmd_id:<wall_time:0 random:0 > key:"\000\000meta2a" user:"root" r
I0408 15:41:13.522449     271 multiraft.go:644] Committed Entry[4]: 6/26 EntryNormal 00000000000000004e25865f44fa13d1: raft_id:1 cmd:<internal_resolve_intent:<header:<timestamp:<wall_time:0 logical:17 > cmd_id:<wall_time:0 random:0 > key:"\000\000meta2\377\377" user:"
I0408 15:41:13.523301     271 multiraft.go:644] Committed Entry[5]: 6/27 EntryNormal 0000000000000000131339e37bcd83b5: raft_id:1 cmd:<internal_resolve_intent:<header:<timestamp:<wall_time:0 logical:17 > cmd_id:<wall_time:0 random:0 > key:"\000range-tree-root" user:"ro
--- FAIL: TestStoreRangeSplitConcurrent (0.17s)
	<autogenerated>:31: Test appears to have leaked an rpc client:
		net.(*pollDesc).Wait(0xc208677800, 0x72, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:84 +0x63
		net.(*pollDesc).WaitRead(0xc208677800, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:89 +0x51
		net.(*netFD).Read(0xc2086777a0, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x7f9dba0e1e20, 0xc208681018)
			/usr/src/go/src/net/fd_unix.go:242 +0x4b3
		net.(*conn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0xc208607640, 0x0, 0x0)
			/usr/src/go/src/net/net.go:121 +0x125
		net.(*TCPConn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
--
I0408 15:41:13.674280     271 multiraft.go:638] HardState updated: {Term:6 Vote:257 Commit:25 XXX_unrecognized:[]}
I0408 15:41:13.675222     271 multiraft.go:641] New Entry[0]: 6/25 EntryNormal 13d314291f9f4fff3940d667a3aa3ac2: raft_id:1 cmd:<end_transaction:<header:<timestamp:<wall_time:0 logical:12 > cmd_id:<wall_time:1428507673663328255 random:4125532999287192258 > key:"m
I0408 15:41:13.676168     271 multiraft.go:644] Committed Entry[0]: 6/25 EntryNormal 13d314291f9f4fff3940d667a3aa3ac2: raft_id:1 cmd:<end_transaction:<header:<timestamp:<wall_time:0 logical:12 > cmd_id:<wall_time:1428507673663328255 random:4125532999287192258 > key:"m
I0408 15:41:13.680524     271 raft.go:390] raft: 101 became follower at term 5
I0408 15:41:13.680748     271 raft.go:207] raft: newRaft 101 [peers: [101], term: 5, commit: 10, applied: 10, lastindex: 10, lastterm: 5]
--- FAIL: TestStoreRangeSplit (0.17s)
	<autogenerated>:31: Test appears to have leaked an rpc client:
		net.(*pollDesc).Wait(0xc208677800, 0x72, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:84 +0x63
		net.(*pollDesc).WaitRead(0xc208677800, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:89 +0x51
		net.(*netFD).Read(0xc2086777a0, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x7f9dba0e1e20, 0xc208681018)
			/usr/src/go/src/net/fd_unix.go:242 +0x4b3
		net.(*conn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0xc208607640, 0x0, 0x0)
			/usr/src/go/src/net/net.go:121 +0x125
		net.(*TCPConn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
--
I0408 15:41:14.216927     271 multiraft.go:638] HardState updated: {Term:6 Vote:257 Commit:118 XXX_unrecognized:[]}
I0408 15:41:14.217937     271 multiraft.go:641] New Entry[0]: 6/118 EntryNormal 13d314293ff7114479edf21af6548515: raft_id:2 cmd:<end_transaction:<header:<timestamp:<wall_time:0 logical:239 > cmd_id:<wall_time:1428507674205950276 random:8785944645685511445 > key:
I0408 15:41:14.218865     271 multiraft.go:644] Committed Entry[0]: 6/118 EntryNormal 13d314293ff7114479edf21af6548515: raft_id:2 cmd:<end_transaction:<header:<timestamp:<wall_time:0 logical:239 > cmd_id:<wall_time:1428507674205950276 random:8785944645685511445 > key:
I0408 15:41:14.229301     271 raft.go:390] raft: 101 became follower at term 5
I0408 15:41:14.229524     271 raft.go:207] raft: newRaft 101 [peers: [101], term: 5, commit: 10, applied: 10, lastindex: 10, lastterm: 5]
--- FAIL: TestStoreRangeSplitStats (0.56s)
	<autogenerated>:31: Test appears to have leaked an rpc client:
		net.(*pollDesc).Wait(0xc208677800, 0x72, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:84 +0x63
		net.(*pollDesc).WaitRead(0xc208677800, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:89 +0x51
		net.(*netFD).Read(0xc2086777a0, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x7f9dba0e1e20, 0xc208681018)
			/usr/src/go/src/net/fd_unix.go:242 +0x4b3
		net.(*conn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0xc208607640, 0x0, 0x0)
			/usr/src/go/src/net/net.go:121 +0x125
		net.(*TCPConn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
--
I0408 15:41:15.313958     271 multiraft.go:638] HardState updated: {Term:6 Vote:257 Commit:270 XXX_unrecognized:[]}
I0408 15:41:15.314795     271 multiraft.go:641] New Entry[0]: 6/269 EntryNormal 0000000000000000249c4c120ad92b6f: raft_id:1 cmd:<internal_resolve_intent:<header:<timestamp:<wall_time:0 logical:495 > cmd_id:<wall_time:0 random:0 > key:"\000\000meta2\377\377" user
I0408 15:41:15.315594     271 multiraft.go:641] New Entry[1]: 6/270 EntryNormal 00000000000000003682bdd11f910f6a: raft_id:1 cmd:<internal_resolve_intent:<header:<timestamp:<wall_time:0 logical:495 > cmd_id:<wall_time:0 random:0 > key:"\000range-tree-root" user:"
I0408 15:41:15.316341     271 multiraft.go:644] Committed Entry[0]: 6/269 EntryNormal 0000000000000000249c4c120ad92b6f: raft_id:1 cmd:<internal_resolve_intent:<header:<timestamp:<wall_time:0 logical:495 > cmd_id:<wall_time:0 random:0 > key:"\000\000meta2\377\377" user
I0408 15:41:15.317144     271 multiraft.go:644] Committed Entry[1]: 6/270 EntryNormal 00000000000000003682bdd11f910f6a: raft_id:1 cmd:<internal_resolve_intent:<header:<timestamp:<wall_time:0 logical:495 > cmd_id:<wall_time:0 random:0 > key:"\000range-tree-root" user:"
--- FAIL: TestStoreZoneUpdateAndRangeSplit (1.07s)
	<autogenerated>:31: Test appears to have leaked an rpc client:
		net.(*pollDesc).Wait(0xc208677800, 0x72, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:84 +0x63
		net.(*pollDesc).WaitRead(0xc208677800, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:89 +0x51
		net.(*netFD).Read(0xc2086777a0, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x7f9dba0e1e20, 0xc208681018)
			/usr/src/go/src/net/fd_unix.go:242 +0x4b3
		net.(*conn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0xc208607640, 0x0, 0x0)
			/usr/src/go/src/net/net.go:121 +0x125
		net.(*TCPConn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
--
I0408 15:41:16.055371     271 multiraft.go:641] New Entry[1]: 6/86 EntryNormal 000000000000000075f1b42a29e00edc: raft_id:1 cmd:<internal_resolve_intent:<header:<timestamp:<wall_time:0 logical:267 > cmd_id:<wall_time:0 random:0 > key:"\000\000meta2db1" user:"root
I0408 15:41:16.055956     271 multiraft.go:641] New Entry[2]: 6/87 EntryNormal 00000000000000002b976c7e501daf7d: raft_id:1 cmd:<internal_resolve_intent:<header:<timestamp:<wall_time:0 logical:267 > cmd_id:<wall_time:0 random:0 > key:"\000\000meta2db2" user:"root
I0408 15:41:16.056583     271 multiraft.go:644] Committed Entry[0]: 6/85 EntryNormal 000000000000000060e4756f5b62bb85: raft_id:1 cmd:<internal_resolve_intent:<header:<timestamp:<wall_time:0 logical:267 > cmd_id:<wall_time:0 random:0 > key:"\000\000meta1\377\377" user:
I0408 15:41:16.057223     271 multiraft.go:644] Committed Entry[1]: 6/86 EntryNormal 000000000000000075f1b42a29e00edc: raft_id:1 cmd:<internal_resolve_intent:<header:<timestamp:<wall_time:0 logical:267 > cmd_id:<wall_time:0 random:0 > key:"\000\000meta2db1" user:"root
I0408 15:41:16.057822     271 multiraft.go:644] Committed Entry[2]: 6/87 EntryNormal 00000000000000002b976c7e501daf7d: raft_id:1 cmd:<internal_resolve_intent:<header:<timestamp:<wall_time:0 logical:267 > cmd_id:<wall_time:0 random:0 > key:"\000\000meta2db2" user:"root
--- FAIL: TestStoreRangeSplitOnConfigs (0.75s)
	<autogenerated>:31: Test appears to have leaked an rpc client:
		net.(*pollDesc).Wait(0xc208677800, 0x72, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:84 +0x63
		net.(*pollDesc).WaitRead(0xc208677800, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:89 +0x51
		net.(*netFD).Read(0xc2086777a0, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x7f9dba0e1e20, 0xc208681018)
			/usr/src/go/src/net/fd_unix.go:242 +0x4b3
		net.(*conn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0xc208607640, 0x0, 0x0)
			/usr/src/go/src/net/net.go:121 +0x125
		net.(*TCPConn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
--
I0408 15:41:16.281744     271 multiraft.go:644] Committed Entry[0]: 6/45 EntryNormal 13d31429bb4285473e2b9e5941e1fad2: raft_id:1 cmd:<put:<header:<timestamp:<wall_time:0 logical:39 > cmd_id:<wall_time:1428507676274492743 random:4479848360544697042 > key:"\000\000meta2
I0408 15:41:16.283321     271 multiraft.go:633] node 257: group 1 raft ready
I0408 15:41:16.283411     271 multiraft.go:638] HardState updated: {Term:6 Vote:257 Commit:46 XXX_unrecognized:[]}
I0408 15:41:16.283934     271 multiraft.go:641] New Entry[0]: 6/46 EntryNormal 13d31429bb42ec9c0d4476bebf1ae7ab: raft_id:1 cmd:<put:<header:<timestamp:<wall_time:0 logical:40 > cmd_id:<wall_time:1428507676274519196 random:956019582531463083 > key:"\000\000meta1\
I0408 15:41:16.284493     271 multiraft.go:644] Committed Entry[0]: 6/46 EntryNormal 13d31429bb42ec9c0d4476bebf1ae7ab: raft_id:1 cmd:<put:<header:<timestamp:<wall_time:0 logical:40 > cmd_id:<wall_time:1428507676274519196 random:956019582531463083 > key:"\000\000meta1\
--- FAIL: TestUpdateRangeAddressing (0.22s)
	<autogenerated>:31: Test appears to have leaked an rpc client:
		net.(*pollDesc).Wait(0xc208677800, 0x72, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:84 +0x63
		net.(*pollDesc).WaitRead(0xc208677800, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:89 +0x51
		net.(*netFD).Read(0xc2086777a0, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x7f9dba0e1e20, 0xc208681018)
			/usr/src/go/src/net/fd_unix.go:242 +0x4b3
		net.(*conn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0xc208607640, 0x0, 0x0)
			/usr/src/go/src/net/net.go:121 +0x125
		net.(*TCPConn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
--
			/go/src/github.com/cockroachdb/cockroach/storage/main_test.go:29 +0x36
		main.main()
			github.com/cockroachdb/cockroach/storage/_test/_testmain.go:332 +0x28d
=== RUN TestUpdateRangeAddressingSplitMeta1
I0408 15:41:16.381585     271 multiraft.go:407] node 257 starting
--- FAIL: TestUpdateRangeAddressingSplitMeta1 (0.10s)
	<autogenerated>:31: Test appears to have leaked an rpc client:
		net.(*pollDesc).Wait(0xc208677800, 0x72, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:84 +0x63
		net.(*pollDesc).WaitRead(0xc208677800, 0x0, 0x0)
			/usr/src/go/src/net/fd_poll_runtime.go:89 +0x51
		net.(*netFD).Read(0xc2086777a0, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x7f9dba0e1e20, 0xc208681018)
			/usr/src/go/src/net/fd_unix.go:242 +0x4b3
		net.(*conn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0xc208607640, 0x0, 0x0)
			/usr/src/go/src/net/net.go:121 +0x125
		net.(*TCPConn).Read(0xc2085fe040, 0xc2085a7000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
--
			/go/src/github.com/cockroachdb/cockroach/util/leaktest/leaktest.go:34 +0x36
		github.com/cockroachdb/cockroach/storage.TestMain(0xc208030b90)
			/go/src/github.com/cockroachdb/cockroach/storage/main_test.go:29 +0x36
		main.main()
			github.com/cockroachdb/cockroach/storage/_test/_testmain.go:332 +0x28d
FAIL
FAIL	github.com/cockroachdb/cockroach/storage	10.023s
=== RUN TestBatchBasics
--- PASS: TestBatchBasics (0.00s)
=== RUN TestBatchGet
--- PASS: TestBatchGet (0.00s)
=== RUN TestBatchMerge
--- PASS: TestBatchMerge (0.00s)
=== RUN TestBatchProto
--- PASS: TestBatchProto (0.00s)
=== RUN TestBatchScan
--- PASS: TestBatchScan (0.00s)

```
Please assign, take a look and update the issue accordingly.

--- a/util/leaktest/leaktest.go
+++ b/util/leaktest/leaktest.go
@@ -117,7 +117,7 @@ func AfterTest(t testing.TB) {
 		"(*Range).AddCmd":                              "a range command",
 	}
 	var stacks string
-	for i := 0; i < 4; i++ {
+	for i := 0; i < 8; i++ {
 		bad = ""
 		stacks = strings.Join(interestingGoroutines(), "\n\n")
 		for substr, what := range badSubstring {

